github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/config.groovy

```diff
@@ -27,7 +27,7 @@ f.block {
             f.checkbox()
         }
 
-        f.entry(title: "Skip first run", field: "skipFirstRun") {
+        f.entry(title: "Ignore older PRs", field: "skipFirstRun") {
```

Comment:
That not really ignore PRs, but skip builds for the "check run". Logic will save PRs state, but wouldn't trigger builds for it. Second check will continue work as expected. So this toggle will be disabled automatically as soon as it do first run.